packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2850,24 +2842,61 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }
     }
 
+    private async waitForDeltaManagerToCatchup(latestSnapshotRefSeq: number,
+        summaryLogger: ITelemetryLogger,
+        ): Promise<void> {
+        if (latestSnapshotRefSeq > this.deltaManager.lastSequenceNumber) {
+            // We need to catch up to the latest summary's reference sequence number before pausing.
```

Comment:
Update this comment - it's not before pausing anymore since this is a generic function.